airbyte-cdk/python/unit_tests/sources/file_based/scenarios/parquet_scenarios.py

```diff
@@ -629,3 +630,34 @@
         }
     )
 ).build()
+
+parquet_with_invalid_config_scenario = (
+    TestScenarioBuilder()
+    .set_name("parquet_with_invalid_config_scenario")
+    .set_config(
+        {
+            "streams": [
+                {
+                    "name": "stream1",
+                    "file_type": "parquet",
+                    "globs": ["*"],
+                    "validation_policy": "emit_record",
+                    "format": {
+                       "parquet": {
+                           "filetype": "csv",
+                       }
+                    }
+                }
+            ]
+        }
+    )
+    .set_stream_reader(TemporaryParquetFilesStreamReader(files=_single_parquet_file, file_type="parquet"))
+    .set_file_type("parquet")
+    .set_expected_read_error(ConfigValidationError, "Error creating stream config object. Contact Support if you need assistance.")
```

Comment:
I guess this is invalid because of the csv, right? Could we have a better error message? Also, could this be a unit test?